From 3af7820cffdd06a42cfd7a9f3e36cb4f423706f9 Mon Sep 17 00:00:00 2001
From: Vinu Vaghasia <vinu.vaghasia@amd.com>
Date: Fri, 14 Jan 2022 14:11:23 -0600
Subject: [PATCH 1/1] linux/driver/hwmon: Modify hwmon driver for dbg msg

dev_info() debug messages added with hwmon drivers that helps BMC boot time
debugging

Signed-off-by: Vinu Vaghasia <vinu.vaghasia@amd.com>
---
 drivers/hwmon/hwmon.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/hwmon/hwmon.c b/drivers/hwmon/hwmon.c
index 6c684058bfdf..0692fbab9c3d 100644
--- a/drivers/hwmon/hwmon.c
+++ b/drivers/hwmon/hwmon.c
@@ -953,6 +953,9 @@ devm_hwmon_device_register_with_info(struct device *dev, const char *name,

 	hwdev = hwmon_device_register_with_info(dev, name, drvdata, chip,
 						groups);
+
+	dev_info(dev, "%s: sensor '%s'\n", dev_name(hwdev), name);
+
 	if (IS_ERR(hwdev))
 		goto error;

--
2.17.1

